web/cypress/e2e/business-formation.spec.ts

```diff
@@ -254,12 +254,12 @@ const selectAndTypeRegisteredAgent = ({
         .should("contain", agentOfficeAddressLine2);
     }
 
-    if (agentOfficeAddressMunicipality) {
-      onBusinessFormationPage.selectRegisteredAgentMunicipality(agentOfficeAddressMunicipality.displayName);
+    if (agentOfficeAddressCity) {
+      onBusinessFormationPage.typeRegisteredAgentMunicipality(agentOfficeAddressCity);
```

Comment:
pebble - update the function name - `typeRegisteredAgentMunicipality` is not correct b/c we removed Municipality

Reply:
I made this change. Perhaps it changed back when I rebased.